### ISSUE TITLE
be able to search sad servers scenarios

### ISSUE THREAD

**Issue by sibelius** [fduran/sadservers#55]
add a basic search tofind sad servers scenarios

**Comment by fduran** [comment]
Yes. Right now there's the tags but yes as the number of scenarios grow, not only some basic search may be needed but also some sort of pagination or collapsing scenario entries.
